cmd/controller-manager/app/controllermanager.go

```diff
@@ -146,7 +169,6 @@ func startClusterController(ctx ControllerContext) (enabled bool, err error) {
 		ClusterStartupGracePeriod: opts.ClusterStartupGracePeriod.Duration,
 	}
 	if err := clusterController.SetupWithManager(mgr); err != nil {
-		klog.Fatalf("Failed to setup cluster controller: %v", err)
```

Comment:
Why do you think we should remove these logs?

Reply:
This is really unnecessary cause we can do this uniformly in `StartControllers` like I mentioned in https://github.com/karmada-io/karmada/pull/1093#discussion_r768272821

Reply:
Yeah, I'm sold.